examples/lines_bars_and_markers/fill.py

```diff
@@ -76,18 +76,12 @@ def _koch_snowflake_complex(order):
 
 plt.show()
 
-#############################################################################
+###############################################################################
 #
-# ------------
+# .. admonition:: References
 #
-# References
-# """"""""""
+#    The use of the following functions, methods, classes and modules is shown
+#    in this example:
 #
-# The use of the following functions, methods, classes and modules is shown
-# in this example:
-
-import matplotlib
-matplotlib.axes.Axes.fill
-matplotlib.pyplot.fill
-matplotlib.axes.Axes.axis
-matplotlib.pyplot.axis
+#    - `matplotlib.axes.Axes.fill` / `matplotlib.pyplot.fill`
```

Comment:
I'll block because I don't think this creates the backlinks in `matplotlib.axes.Axes.fill` that are the point of this block... (https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.fill.html#examples-using-matplotlib-axes-axes-fill)